src/sentry/spans/consumers/process/factory.py

```diff
@@ -58,45 +60,89 @@ def process_message(message: Message[KafkaPayload]) -> ProduceSegmentContext | N
         return None
 
     assert isinstance(message.value, BrokerValue)
-    try:
-        span = _deserialize_span(message.payload.value)
+    metrics.incr("process_spans.spans.received.count")
+
+    with sentry_sdk.start_transaction(op="process", name="spans.process.process_message") as txn:
```

Comment:
These transactions should go to s4s, they're marked as unsafe
https://github.com/getsentry/sentry/blob/63b5957ac7d8530b08534f1e88ad08ca801fa618/src/sentry/utils/sdk.py#L39-L40